internal/pkg/template/diff/lcs.go

```diff
@@ -0,0 +1,53 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package diff
+
+type eqFunc func(inA, inB int) bool
```

Comment:
Why do we need this 🤔

Reply:
It doesn't matter, just for easier refactoring (potentially). For example, if we want to change the `eq` parameter to `func(inA, inB) (bool, error)`, we won't need to change the function signature everywhere - we just need to change the code around where it's invoked.